pkg/minikube/audit/entry.go

```diff
@@ -17,16 +17,20 @@ limitations under the License.
 package audit
 
 import (
+	"bytes"
+	"encoding/json"
+	"fmt"
 	"time"
 
+	"github.com/olekukonko/tablewriter"
 	"github.com/spf13/viper"
 	"k8s.io/minikube/pkg/minikube/config"
 	"k8s.io/minikube/pkg/minikube/constants"
 )
 
 // entry represents the execution of a command.
 type entry struct {
```

Comment:
choose better name... is this representing one row ?

Reply:
I renamed it to `singleEntry` if you have a better idea lmk